parser.py

```diff
@@ -510,13 +250,16 @@ def uop(self, *args):
             # <id> <anylist> , <uop>
             # <id>() <anylist>, <uop>
             # <id>(<explist>) <anylist>, <uop>
-            op = Op(op_name, params, wires)
+            # This case only appears for tensor products of operators
+            # TODO: consider extending the grammar so users can specify
+            #  "simultaneous" gates/channels via a tensor product
+            op = Op(Ops.OPERATOR, op_name, params, wires)
             return TensorOp(op, other_op)
 
-        if extra_args[0].name == "idlist":
+        if extra_args[0].name == Lists.IDLIST:
             # TODO: this case should not occur, according
             # to the original spec. Determine what is causing it
-            # @co9olguy: this is because self.gatedecl calls
+            # @josh146: this is because self.gatedecl calls
```

Comment:
don't want this comment to be attributed to me :laughing:

Reply:
😆 this comment block reminds me of
![image](https://user-images.githubusercontent.com/2959003/97274348-7fd47a80-186f-11eb-9c79-dd83c6dec94e.png)